svthalia/concrexit
# Implement app-based user menu

Closes #3158 



### Summary
This PR separates out the user menu so that entries are stored in the apps, just like the general menu.
Additionally, the currently active page in the user menu is highlighted, again just like the general menu.

### How to test
Steps to test the changes you made:
1. Go to a page
2. See that the menu still contains all entries
3. Click a link in the menu
4. See that it colors magenta
